core/templates/dev/head/css/oppia.css

```diff
@@ -15,6 +15,10 @@
   Animation keyframes. These must be placed at the top of the file
   in order to work in IE.
 */
+*, *:before, *:after {
```

Comment:
You should not be defining properties that apply to all elements. Try and do this in a more targeted way. We have had multiple breakages before due to overly-broad CSS rules.

Reply:
Done, I've added to md-button it was needed for them!